FtoC.java

```diff
@@ -1,13 +1,25 @@
 class FtoC extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * A program FtoC.java that receives a degree in Fahrenheit from the user
+  * and prints the result in celsius.
+  * @author: Kayla Jin
   */
   
   public void run() {
     
-    // start coding here
+    // Declare variables
+    double dblFahrenheit;
+    double dblCelsius;
+
+    // Receive user input
+    dblFahrenheit = readDouble("Enter the degrees in fahrenheit: ");
+
+    // Calculate the degrees in celsius
+    dblCelsius = 5.0 / 9.0 * (dblFahrenheit-32);
```

Comment:
The solution uses spaces on both sides of the subtraction sign of (dblFahrenheit-32), however, this code does not have spaces on both sides of the subtraction sign to highlight the order of operations.